src/pages/edit-contact/edit-contact.ts

```diff
@@ -72,6 +72,8 @@ export class EditContactPage {
 
   handleSubmit() {
     var toastMessage;
```

Comment:
O.o' 
a var in typescript

Reply:
technically, it's not a problem, but it's not good standards

Reply:
NO! That's just bad, no var in typescript.

Reply:
fixed